ui/src/containers/organisations.list/users/index.tsx

```diff
@@ -1,18 +1,58 @@
 import { DataTable, EmptyState, Flex } from "@raystack/apsara";
 import { useFrontier } from "@raystack/frontier/react";
-import { useEffect, useState } from "react";
-import { useOutletContext, useParams } from "react-router-dom";
+import { useCallback, useEffect, useState } from "react";
+import { Outlet, useOutletContext, useParams } from "react-router-dom";
 
-import { V1Beta1Organization, V1Beta1User } from "@raystack/frontier";
+import {
+  V1Beta1ListOrganizationUsersResponseRolePair,
+  V1Beta1Organization,
+  V1Beta1Role,
+  V1Beta1User,
+} from "@raystack/frontier";
 import { OrganizationsHeader } from "../header";
 import { getColumns } from "./columns";
+import { reduceByKey } from "~/utils/helper";
+import * as R from "ramda";
 
 type ContextType = { user: V1Beta1User | null };
+
 export default function OrganisationUsers() {
   const { client } = useFrontier();
-  let { organisationId } = useParams();
+  let { organisationId, userId } = useParams();
   const [organisation, setOrganisation] = useState<V1Beta1Organization>();
-  const [users, setOrgUsers] = useState([]);
+  const [users, setOrgUsers] = useState<V1Beta1User[]>([]);
+  const [rolePairs, setRolePairs] = useState<
+    V1Beta1ListOrganizationUsersResponseRolePair[]
+  >([]);
+  const [isRolesLoading, setIsRolesLoading] = useState(false);
+  const [roles, setRoles] = useState<V1Beta1Role[]>([]);
+
+  const getRoles = useCallback(
+    async (ordId: string) => {
+      try {
+        setIsRolesLoading(true);
+
+        const {
+          // @ts-ignore
+          data: { roles: orgRoles },
+        } = await client?.frontierServiceListOrganizationRoles(ordId, {
+          scopes: ["app/organization"],
+        });
+        const {
+          // @ts-ignore
+          data: { roles },
+        } = await client?.frontierServiceListRoles({
+          scopes: ["app/organization"],
+        });
+        setRoles([...roles, ...orgRoles]);
```

Comment:
Just confirming, this current PR adds functionality just to update org level roles, right?

Reply:
yes only org level roles